src/services/dealer.ts

```diff
@@ -198,3 +198,29 @@ export const putDealerPromotion = async ({
     return handleValidationError(error, { swallowErrors: true })
   }
 }
+
+export const requestMatelsoIntegration = async ({
+  dealerId,
+  options = {},
+}: {
+  dealerId: number
+  options?: ApiCallOptions
+}): Promise<WithValidationError> => {
+  try {
+    const result = await postData({
+      path: `dealers/${dealerId}/request-matelso-integration`,
+      body: {},
+      options: {
+        isAuthorizedRequest: true,
+        ...options,
+      },
+    })
+
+    return {
+      tag: "success",
+      result,
+    }
+  } catch (error) {
+    return handleValidationError(error, { swallowErrors: true })
```

Comment:
What validation errors are we getting from that endpoint?

Reply:
none, this is the default one that I use in case BE returns some errors, what should I use instead?

Reply:
As far as I have in mind this is used to handle / format filed validation errors when sending data to the API for format the validation messages and since for this call I would the API expect to just succeed or reject so I don't see why we would need to add form validation handling here...